database/migrations/2024_12_10_014452_create_participant_healthy_marriage_introductory_survey.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('participant_healthy_marriage_introductory_survey', function (Blueprint $table) {
```

Comment:
We may need to add _and_responsible_fatherhood to this survey and the assessment.  Alternatively, we could simplify it to participant_marriage_and_fatherhood_survey (same for assessment).